docs/src/devdocs.md

```diff
@@ -23,3 +23,14 @@ In principle, the following should be done:
 1. Extend `nearby_ids(position, model, r)`.
 
 And that's it!
+
+## Designing a new Pathfinder Cost Metric
+
+To define a new cost metric, simply make a struct that subtypes `CostMetric` and provide
+a `delta_cost` function for it. These methods work for [`AStar`](@ref) at present, but
+will be available for other pathfinder algorithms in the future.
+
+```@docs
+CostMetric
+delta_cost
+```
```

Comment:
This is not enough. You must tell users what **methods** should be extended by their new cost metric.

Reply:
I don't quite follow this requirement. What is the method that needs to be extended other than `delta_cost`?

Reply:
It's specified in the `delta_cost` docstring that `delta_cost(pathfinder, metric, from, to)` needs to be extended. Should that be moved/mentioned here too?

Reply:
Sorry, my fault. I haven't looked in detail the docstrings in the expanded docs. Consider this resolved!